### PR TITLE
Using localstorage to modify clients APIs for front-end development

### PR DESCRIPTION
**Pull request by Leowisd**
This implements oncokb/oncokb#2134

For local dev, it can use `localStorage.setItem('localdev', 'SOME_URL')` to enable connection.

### CODE REVIEW

File: src/main/webapp/app/shared/api/clientInstance.ts [hunk 1]
```diff
@@ -1,5 +1,8 @@
 import API from './generated/API';
 
-const client = new API();
+let devUrl = localStorage.getItem('localdev');
```
Comment (zhx828) [inline src/main/webapp/app/shared/api/clientInstance.ts:3]: Let's put the localdev into the constants.ts

File: src/main/webapp/app/shared/api/clientInstance.ts [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
+if (devUrl === null) devUrl = '';
```
Comment (zhx828) [inline src/main/webapp/app/shared/api/clientInstance.ts:4]: `let devUrl = localStorage.getItem('localdev') || ''`

File: src/main/webapp/app/shared/api/oncokbClientInstance.ts [hunk 3]
```diff
@@ -1,5 +1,9 @@
 import OncoKbAPI from './generated/OncoKbAPI';
 
-const client = new OncoKbAPI('api/v1');
+let devUrl = localStorage.getItem('localdev');
+if (devUrl !== null) devUrl += '/';
+else devUrl = '';
```
Comment (zhx828) [inline src/main/webapp/app/shared/api/oncokbClientInstance.ts:5]: Let's put this to a utils class and reuse in these three files.

**Comment by Leowisd** [comment]
Close this request. I'm going to resubmit this pull request to fix.

**Pull request closed without merging**